modules/monitoring-setting-log-levels-for-monitoring-components.adoc

```diff
@@ -77,8 +80,11 @@ data:
     <component>: <1>
       logLevel: <log_level> <2>
 ----
-<1> The monitoring component that you are applying a log level to.
-<2> The log level to apply to the component.
+<1> The monitoring stack component for which you are setting a log level.
+For user workload monitoring, available component values are `prometheus`, `prometheusOperator`, and `thanosRuler`.
+<2> The log level to set for the component.
+The available values are `error`, `warn`, `info`, and `debug`. 
+The default value for all components is `info`.
```

Comment:
change to below is better
The default value is `info`.

Reply:
changed